app/main.py

```diff
@@ -2,0 +1,24 @@
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
```

Comment:
According to the checklist, imports should be grouped with a blank line between different groups. The import from `app.cafe` should be separated from the local application imports (`app.errors`).